src/libs/ppscale/ppscale.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2018-2020          Anton Shepelev          (anton.txt@gmail.com) */
+/* Usage of the works is permitted  provided that this instrument is retained */
+/* with the works, so that any entity that uses the works is notified of this */
+/* instrument.    DISCLAIMER: THE WORKS ARE WITHOUT WARRANTY.                 */
+
+/* ----------------------- Pixel-perfect scaling unit ----------------------- */
+/* This unit uses the Horstmann indentation style.                            */
+
```

Comment:
This header is missing guards:

```
#ifndef PPSCALE_H
#define PPSCALE_H

// all declarations

#endif

Reply:
I decided not to use guards because the header exports only functions but no types.
Therefore, it shall be included into source files, but not into other headers. Is it good practice to use an include guard with each and every header? I will ask fellow *C* programmers and consider your proposal.

Reply:
Thanks :)